pkg/policy/l4.go

```diff
@@ -181,36 +183,39 @@ func (l4 *L4Filter) ToKeys(direction trafficdirection.TrafficDirection, identity
 	return keysToAdd
 }
 
-// GetRelevantRules returns the relevant rules based on the source and
-// destination addressing/identity information.
-func (l7 L7DataMap) GetRelevantRules(identity *identity.Identity) api.L7Rules {
-	rules := api.L7Rules{}
-
-	if identity != nil {
-		for selector, endpointRules := range l7 {
-			if selector.Matches(identity.Labels.LabelArray()) {
-				rules.HTTP = append(rules.HTTP, endpointRules.HTTP...)
-				rules.Kafka = append(rules.Kafka, endpointRules.Kafka...)
-				rules.DNS = append(rules.DNS, endpointRules.DNS...)
-				rules.L7Proto = endpointRules.L7Proto
-				rules.L7 = append(rules.L7, endpointRules.L7...)
-			}
-		}
+// IdentitySelectionUpdated implements CachedSelectionUser interface
+func (l4 *L4Filter) IdentitySelectionUpdated(selector CachedSelector, selections, added, deleted []identity.NumericIdentity) {
+	log.Infof("L4Filter::IdentitySelectionUpdated(selector: %v, selections: %v, added: %v, deleted: %v) call received",
```

Comment:
Info --> Debug?

Also recommend using `WithFields` here.

Reply:
Also, this function doesn't do anything because we are creating `L4Filters` and are not doing incremental updates - would be good to note that here, as it's not clear why we aren't doing anything.